backend/Dockerfile

```diff
@@ -15,8 +15,8 @@
 # Build arguments
 ARG SOURCE_CODE=.
 
-#@follow_tag(registry.access.redhat.com/ubi8/go-toolset:1.20)
-FROM registry.access.redhat.com/ubi8/go-toolset:1.20 as builder
+#@follow_tag(registry.access.redhat.com/ubi8/go-toolset:1.21)
```

Comment:
@follow_tags have been removed from midstream, so we should do it here too

Reply:
Done, removed tags.